### ISSUE TITLE
C key doesn't work, won't start gameplay.

### ISSUE THREAD

**Issue by OSUsimp** [KuromeSama6/Autosu#8]
Autosu isn't working. I select a map, go into the menu, and choose all the keys and presets I need. But once I go to turn on the CMD and press C, nothing happens. I attempt it again, using another screen, directly tapping on the task, and still, nothing is happening. However, when I go to exit out, A/P P/RST starts going off. The troubleshooting wiki page doesn't really tell me what I need to do, so that's why I'm posting this.

**Comment by KuromeSama6** [comment]
I hypothesize that it's an issue regarding keyboard hooks and device compatibility, which I have not fully tested for. Maybe the keyboard listeners aren't compatible with your device.

Could you provide more info about your device and operating system, as well as pictures and/or videos of you attempting to use Autosu? This should help me figure things out quicker.

**Comment by OSUsimp** [comment]
> I hypothesize that it's an issue regarding keyboard hooks and device compatibility, which I have not fully tested for. Maybe the keyboard listeners aren't compatible with your device.
> 
> Could you provide more info about your device and operating system, as well as pictures and/or videos of you attempting to use Autosu? This should help me figure things out quicker.


Yes I use a windows surface, and I usually use touch screen to play osu. And I'll send it tommrow once I get on my computer!
